docs: adapt Documentation with new docarray at center (#5941)
jina-ai/jina: docs/cloud-nativeness/docker-compose.md

the Gateway are built with the same Jina version to guarantee compatibility.
````

> ## Example: Index and search text using our own build Encoder and Indexer

Reviewer: "our own build Encoder and Indexer" -- Unless it encodes and indexes builds, this is grammatically wrong, but I don't know exactly what it should be.

Author: own `built` sorry

Author: it is not Jina AIs, is the Executor that each user builds.

Reviewer: If you mean "your own build" then just change it to that and I'll approve.

> Install [`Docker Compose`](https://docs.docker.com/compose/install/) locally to follow this how-to.

Reviewer: ```suggestion
Install [`Docker Compose`](https://docs.docker.com/compose/install/) locally before starting this tutorial.
```

> For this example we recommend to check {ref}`how to build and containerize the Executors to be run in Kubernetes. <build-containerize-for-k8s>`

Reviewer: ```suggestion
For this example we recommend that you read {ref}`how to build and containerize the Executors to be run in Kubernetes. <build-containerize-for-k8s>`
```
